src/styles/blocks/header.scss

```diff
@@ -0,0 +1,56 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
The padding of 0 50px on the header might not be responsive on smaller screens. Consider using relative units or media queries to adjust the padding for different screen sizes.